sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -40,9 +40,17 @@
                 " /></v-col
           ></v-row>
         </content-group>
-        <v-divider></v-divider>
+        <v-divider
+          v-if="
+            restrictionData.isActive &&
+            restrictionData.restrictionCategory !== 'Federal'
```

Comment:
mm, is it 'restrictionType` or `restrictionCategory`

Reply:
yes, it should be restrictionType. Thanks!